src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -17992,5 +17993,184 @@ public void InterpolatedStringInForeach_HasErrors()
                   NextVariables(0)
                 """);
         }
+
+        [Theory, WorkItem("https://github.com/dotnet/roslyn/issues/66235")]
+        [InlineData(@"""""")]
+        [InlineData(@"@""""")]
+        [InlineData(@"""""""test""""""")]
+        public void ConvertStringLiteralToInterpolatedStringHandlerError_ByVal(string stringLiteral)
```

Comment:
Should these be added also to `RawInterpolationTests`?

Reply:
These tests already verify results for different string literals: normal `""`, verbatim `@""` and raw `"""test"""` (see test data). I think moving raw string case out of every test to `RawInterpolationTests` would not bring any value and only split logically grouped test cases up, which isn't good IMO

Reply:
I don't think we need to keep them in sync. Cyrus cloned all the tests to get a good initial test base, but there basically no binding changes between the two syntax forms, the difference is handled entirely during parsing. We have enough coverage that I'm comfortable without adding them.